src/components/ec-metroline-card/ec-metroline-card.story.js

```diff
@@ -0,0 +1,87 @@
+import { boolean } from '@storybook/addon-knobs';
+import { storiesOf } from '@storybook/vue';
+import EcMetrolineCard from './ec-metroline-card.vue';
+import EcIcon from '../ec-icon';
+import EcInputField from '../ec-input-field';
+
+storiesOf('Metroline Card', module)
```

Comment:
We don't use storiesOf since the v6 it's active

Check this [story](https://fxsolutions.atlassian.net/browse/ONL-4998)

Reply:
Updated!